src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -85,11 +102,105 @@ def run(input:str, config: str, rules:str, output: str):
     
     # msdf.prefix_map = sssom_yaml['curie_map']
     # msdf.metadata = sssom_yaml['global_metadata']
+    msdf.df = filter_prefixes(df=msdf.df, filter_prefixes=prefix_of_interest, features=[SUBJECT_ID, OBJECT_ID])
+    # TODO: replace line below by msdf.remove_mappings(mapping_msdf) once imported from SSSOM.
```

Comment:
Shall we make an sssom release and do this, or do you think its not worth waiting?

Reply:
I think we should make a release.